Problem_1.java

```diff
@@ -0,0 +1,32 @@
+/*
+   
+Search in a Rotated Sorted Array (https://leetcode.com/problems/search-in-rotated-sorted-array/)
+ */
+
+//Time comlpexity - O(log n) 
+//space complexity - O(1)
+// Did this code successfully run on Leetcode : Yes
+/*
+Any problem you faced while coding this : None
+
+*/  
+
+// Your code here along with comments explaining your approach
+//two pointer approach where you search the target from the front and back siultaneously
+class Solution {
```

Comment:
This solution is incorrect for the give question. Basically I am not able to see binary search here